### ISSUE TITLE
Language dropdown - unselected items unreadable

### ISSUE THREAD

**Issue by bricefriha** [veganhacktivists/wildanimalsuffering#95]
### Description 📜
There is a visual "bug" when we want to selected a language, the only item visible is the language selected.

### To Reproduce ⚙
Steps to reproduce the behaviour:

Click on the language selection dropdown (top left corner)
Look at all the items.

### Expected behaviour 🕵🏼‍♀️
All the languages should be readable

### Screenshots 💻
![image](https://github.com/veganhacktivists/wildanimalsuffering/assets/37577669/d998e992-c6d7-4413-a8ab-e774ec807ed2)

### Client info:

Device: PC
OS: Windows 11
Browser: Microsoft Edge (Chromium)

### Additional context ℹ
It's not a huge deal, but I think it can be anoying for some people. I can take care of the fix if you want, I just wanted to inform you first 😉

**Comment by stephan281094** [comment]
Hi @bricefriha, thanks for filing this issue. This is definitely something undesirable. Since I'm not on Windows, I can't seem to be able to reproduce this in any browser, however.

I'm curious, maybe this can be solved by setting the `color` to black on select options?
```css
select options {
  color: #000;
}
```

Or like so:
```tsx
<Select defaultValue={locale} onChange={onLangChange}>
  <option className="text-black" value="de">De</option>
  <option className="text-black" value="en">En</option>
  <option className="text-black" value="fr">Fr</option>
</Select>
```

Could you maybe confirm whether this works? Thank you!

**Comment by bricefriha** [comment]
Hey @stephan281094 I'll check this ASAP

**Comment by bricefriha** [comment]
Yep, the second one works neat! 

![image](https://github.com/veganhacktivists/wildanimalsuffering/assets/37577669/7e1f000b-b789-4d86-97ee-1d65a91cf30e)

**Comment by bricefriha** [comment]
Thanks @stephan281094!

**Comment by stephan281094** [comment]
Fixed in #96, this is now fixed on wildanimalsuffering.org. Thanks for checking!
